openapi/paths/aml-checks.yaml

```
@@ -0,0 +1,31 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve a collection of Customer AML checks
```

issue: i think we can shorten this

We need to keep summaries short so that they do not take up a lot of room in the docs left nav. We can provide the detail in the operation description

```suggestion
  summary: Retrieve customer AML checks
```